src/web/components/ApiKeyManagement/KeyItem.tsx

```diff
@@ -16,6 +16,16 @@ type KeyItemProps = {
 };
 function KeyItem({ apiKey: apiKeyInitial, onEdit, onDisable, availableRoles }: KeyItemProps) {
   const [apiKey, setApiKey] = useState<ApiKeyDTO>(apiKeyInitial);
+  const [showKeyDisableDialog, setShowKeyDisableDialog] = useState<boolean>(false);
+  const [showKeyEditDialog, setShowKeyEditDialog] = useState<boolean>(false);
```

Comment:
Is there a way to combine these or is is best to leave it as two separate lines?

Reply:
you could probably be tricky and store an object in state.  like { firstDialog: true, secondDialog: false }, but that seems more complicated than helpful.